piqueserver/server.py

```diff
@@ -248,7 +261,7 @@ def __init__(self, interface, config):
         self.max_players = config.get('max_players', 20)
         self.melee_damage = config.get('melee_damage', 100)
         self.max_connections_per_ip = config.get('max_connections_per_ip', 0)
-        self.passwords = config.get('passwords', {})
+        self.passwords = config_store.option('passwords', default={}, validate=check_passwords).get()
```

Comment:
Note that this causes an error instead of just a warning

Reply:
What error does it cause?

Reply:
ẁell, in the old version it would just print a warning about default passwords. With this code, afaict, it would fail with a CommandError if any default passwords were present.

Reply:
That function returns `True`, so it will always pass the validation. I just added some clarifying comments to the code. :)